presidio-structured/presidio_structured/analysis_builder.py

```diff
@@ -159,6 +159,7 @@ def generate_analysis(
         df: DataFrame,
         n: Optional[int] = None,
         language: str = "en",
+        selection_strategy: str = "most_common"
```

Comment:
Please add the selection strategies to the docstring (you have that in the inner function, could be copied here as well)